Sources/Commands/SwiftPackageTool.swift

```diff
@@ -290,22 +290,41 @@ extension SwiftPackageTool {
     
     struct APIDiff: SwiftCommand {
         static let configuration = CommandConfiguration(
-            commandName: "experimental-api-diff")
+            commandName: "experimental-api-diff",
+            abstract: "Diagnose API-breaking changes to Swift modules in a package",
+            discussion: """
+            The experimental-api-diff command can be used to compare the Swift API of \
+            a package to a baseline revision, diagnosing any breaking changes which have \
+            been introduced. By default, it compares every Swift module from the baseline \
+            revision which is part of a library product. If this behavior is undesirable, \
+            the `--product` and `--target` options can be used to restrict the scope of \
+            the comparison.
+            """)
 
         @OptionGroup(_hiddenFromHelp: true)
         var swiftOptions: SwiftToolOptions
 
         @Argument(help: "The baseline treeish to compare to (e.g. a commit hash, branch name, tag, etc.)")
         var treeish: String
 
+        @Option(name: .customLong("product"), help: "One or more products to include in the diff")
```

Comment:
Let's use a plural "products" since there can be multiple.

Reply:
👍 changed, I also switched the parsing strategy so these can be passed as `--products P1 P2` instead of `--product P1 --product P2`.